ui/app/src/model/config-client.ts

```diff
@@ -52,3 +58,29 @@ export function useIsReadonly() {
   }
   return data.readonly;
 }
+
+export function useDashboardLists() {
```

Comment:
to avoid a confusion with the function coming from `dashboard-client.ts` I would rename it `useConfigDashboardLists`